### PR TITLE
Translations - Add Simplified Chinese

### PR DESCRIPTION
**Pull request by PlayerBotPro**
**When merged this pull request will:**
- _Add translation of Simplified Chinese_
- _Each change in a separate line_

### IMPORTANT

- If the contribution affects [the documentation](https://github.com/acemod/ACE3/tree/master/docs), please include your changes in this pull request so the documentation will appear on the [website](https://ace3.acemod.org/).
- [Development Guidelines](https://ace3.acemod.org/wiki/development/) are read, understood and applied.
- Title of this PR uses our standard template `Component - Add|Fix|Improve|Change|Make|Remove {changes}`.

### CODE REVIEW

**Review by PabstMirror — APPROVED** [review]

**Comment by acknowledge14** [comment]
@PlayerBotPro
1 “强迫另一个炮塔转向你正在观察的位置”
这里换成“强制”会不会更好？

2 override 现在翻译的是“超控”，如果换成“超越射击”，会不会更好？

File: addons/hunterkiller/stringtable.xml [hunk 1]
```diff
@@ -36,6 +38,7 @@
             <German>Richtet deinen Turm auf die Blickrichtung eines anderen Turms</German>
             <Korean>포탑을 다른 포탑의 시야에 놓습니다.</Korean>
             <Japanese>自分のタレットを相手のタレットの視点に旋回させる。</Japanese>
+            <Chinesesimp>将你的炮塔转向至另一个炮塔正在观察的位置</Chinesesimp>
```
Comment (PabstMirror) [inline addons/hunterkiller/stringtable.xml:41]: these seem to be the referenced lines, feel free to make a replacement suggestion

File: addons/hunterkiller/stringtable.xml [hunk 2]
```diff
@@ -48,6 +51,7 @@
             <German>Überschreibe</German>
             <Korean>오버라이드</Korean>
             <Japanese>オーバーライド</Japanese>
+            <Chinesesimp>超控</Chinesesimp>
```
Comment (PabstMirror) [inline addons/hunterkiller/stringtable.xml:54]: these seem to be the referenced lines, feel free to make a replacement suggestion

Comment (PlayerBotPro) [inline addons/hunterkiller/stringtable.xml:54]:
```suggestion
            <Chinesesimp>超越控制</Chinesesimp>
```

File: addons/hunterkiller/stringtable.xml [hunk 3]
```diff
@@ -60,6 +64,7 @@
             <German>Zwingt andere Türme, sich auf deine Blickrichtung zu drehen</German>
             <Korean>다른 포탑이 시야에 오도록 강제합니다.</Korean>
             <Japanese>他のタレットを強制的に自分の視点に旋回させる</Japanese>
+            <Chinesesimp>强迫另一个炮塔转向你正在观察的位置</Chinesesimp>
```
Comment (PlayerBotPro) [inline addons/hunterkiller/stringtable.xml:67]:
```suggestion
            <Chinesesimp>强制另一个炮塔转向你正在观察的位置</Chinesesimp>
```

File: addons/hunterkiller/stringtable.xml (hunk 2 above)
Comment (PlayerBotPro) [reply addons/hunterkiller/stringtable.xml:54]:
```suggestion
            <Chinesesimp>超越控制</Chinesesimp>
```

**Comment by PlayerBotPro** [comment]
> @PlayerBotPro 1 “强迫另一个炮塔转向你正在观察的位置” 这里换成“强制”会不会更好？
> 
> 2 override 现在翻译的是“超控”，如果换成“超越射击”，会不会更好？

1 改为"强制"显然更好, 感谢建议

2 当前版本的override只有强制炮塔转向的功能, 无法完成超越射击的功能. 已将"超控"改为"超越控制", 以更准确的描述用途

File: addons/hunterkiller/stringtable.xml (hunk 1 above)
Comment (PlayerBotPro) [reply addons/hunterkiller/stringtable.xml:41]: Its in another line, changed it down below

**Review by PabstMirror — APPROVED** [review]

**Pull request merged**